### ISSUE TITLE
Does this work with braintree-web version 3?

### ISSUE THREAD

**Issue by somghosh** [jeffcarp/braintree-react#33]
It seems that it does not work with the braintree-web version 3 , as I see in the package.json of this git repo 

    "braintree-web": "^2.9.0",

**Comment by kyledetella** [comment]
v3 of `braintree-web` takes a _very_ different approach to configuration and setup than v2. It also does not include Drop-in. As such, this library doesn't yet support v3.

It would be awesome to look into adding support for it though.

@jeffcarp what do you think?

**Comment by jeffcarp** [comment]
Yeah that would be awesome! But as @kyledetella said since there is no v3 version of Drop-In available it isn't a possibility right now.

**Comment by toddmedema** [comment]
Following up here, it seems like there is a v3 available now? https://developers.braintreepayments.com/guides/drop-in/javascript/v3

**Comment by jeffcarp** [comment]
It is! But this module [is no longer maintained](https://github.com/jeffcarp/braintree-react/blob/master/README.md). Feel free to fork/remix/etc.
